scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -71,6 +72,11 @@ class FormatWriter(formatOps: FormatOps) {
       }
 
       entry.formatWhitespace(sb)
```

Comment:
just curious: will this introduce any (unwanted) alignment here if `format:off` had been used?

Reply:
Good catch, fixed in https://github.com/scalameta/scalafmt/pull/1773/commits/1589507b3df085fa80eb5ac28210eb9fe4307406

Reply:
Sorry, didn't occur to me before: what about multiple spaces in the original code? Perhaps we should just add the contents of FormatToken.between?

Reply:
Added test for multiple spaces https://github.com/scalameta/scalafmt/pull/1773/commits/3185a373616ac507bb0fe4254370307b09379493 Or did you mean other cases?